app/services/contracts.server.ts

```diff
@@ -1,44 +1,41 @@
-
 export function getContracts({ network }: { network: string }) {
```

Comment:
This file is just formatting - I believe @masonchain had a different tab setting updated prettierrc to override individual ones in other PR that has been merged

Reply:
I realize you didn't change anything here, but wondering what you think about switching this to use the `import` syntax. It gives up types and gives you an IDE error if the contracts are missing from `core-evm-contracts`.
https://github.com/MetricsDAO/xyz/blob/c2a74ced6f379c445bc1aca255908219aa0fbdf0/app/services/contracts.server.ts

Reply:
You could I always thought this was cleaner and easier to read - but I originally had it like this and it works locally just not on deployment which was frustrating
```
  xMetricJson = require(`core-evm-contracts/deployments/${network}/Xmetric.json`);
  questionAPIJson = require(`core-evm-contracts/deployments/${network}/QuestionAPI.json`);
  questionStateController = require(`core-evm-contracts/deployments/${network}/QuestionStateController.json`);
  bountyQuestionJson = require(`core-evm-contracts/deployments/${network}/BountyQuestion.json`);
  costController = require(`core-evm-contracts/deployments/${network}/ActionCostController.json`);
  vaultJson = require(`core-evm-contracts/deployments/${network}/Vault.json`);
  ```

Reply:
Just tried it again weird how it can work locally but not on deployment - https://stackoverflow.com/questions/72239792/how-to-use-dynamic-require-in-remix

I say keep it like this for now and we can refactor again if you like, I'm hoping somehow dynamic imports will be supported in future